test/standard/formatter_test.rb

```diff
@@ -84,8 +84,8 @@ def test_prints_rake_message
   end
 
   def test_prints_call_for_feedback
-    @subject.file_finished(@some_path, [Offense.new(false, 42, 13, "Neat")])
-    @subject.file_finished(@some_path, [Offense.new(false, 43, 14, "Super")])
+    @subject.file_finished(@some_path, [Offense.new(false, 42, 13, "Neat", "Bundler/InsecureProtocolSource")])
+    @subject.file_finished(@some_path, [Offense.new(false, 43, 14, "Super", "Bundler/InsecureProtocolSource")])
```

Comment:
Would it be better to just address the root cause by doing whatever it wants us to do to Standard's Gemfile than to churn these tests in an incidental way?

Reply:
@searls I'm sorry, but I didn't fully understand what you meant. Could you please rephrase it?

From my point of view: these tests are getting churned because we're passing in a stubbed version of the `Offense` class. 
Since `Formatter#file_finished` now checks if the offense is autocorrectable (through the offense's `cop_name`), in the tests we must suply a stub which responds to the `cop_name` method.

I don't understand how doing anything to Standard's Gemfile would solve this issue.